File: app/src/main/java/me/zipi/navitotesla/ui/favorite/FavoriteViewModel.java
package me.zipi.navitotesla.ui.favorite;

import java.util.ArrayList;
import java.util.List;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;
import lombok.Getter;
import me.zipi.navitotesla.db.PoiAddressEntity;

public class FavoriteViewModel extends ViewModel {

    @Getter
    private final MutableLiveData<List<PoiAddressEntity>> recentPoiAddress = new MutableLiveData<>(new ArrayList<>());
    @Getter
    private final MutableLiveData<List<PoiAddressEntity>> registeredPoiAddress = new MutableLiveData<>(new ArrayList<>());


}

File: app/src/main/kotlin/me/zipi/navitotesla/ui/favorite/FavoriteViewModel.kt
package me.zipi.navitotesla.ui.favorite

import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import me.zipi.navitotesla.db.PoiAddressEntity

class FavoriteViewModel : ViewModel() {
    val recentPoiAddress = MutableLiveData<MutableList<PoiAddressEntity>>()

    val registeredPoiAddress = MutableLiveData<MutableList<PoiAddressEntity>>()
}
